### PR TITLE
DATA-854 remote geometry functionality

### PR DESCRIPTION
**Pull request by biotinker**
Implements the API updates made here:

https://docs.google.com/document/d/1DXv1qc1MtK_OLaCNFMuC8BnA6elGJ1iLewgZId99MfY/edit

### CODE REVIEW

**Comment by biotinker** [comment]
@raybjork @tessavitabile  Note that this PR will have to be merged manually after approval by Eric or James, as this was a breaking change, and CI does not allow those to be merged without extra eyes.

File: proto/viam/common/v1/common.proto
Comment (tessavitabile) [inline proto/viam/common/v1/common.proto:68]: Should this have a comment describing its purpose?

**Review by tessavitabile — APPROVED** [review]

File: proto/viam/common/v1/common.proto
Comment (biotinker) [reply proto/viam/common/v1/common.proto:68]: Yes, definitely; I will add.

**Review by raybjork — APPROVED** [review]

File: proto/viam/common/v1/common.proto
Comment (tessavitabile) [inline proto/viam/common/v1/common.proto:134]: The scope says this will be optional.

Comment (biotinker) [reply proto/viam/common/v1/common.proto:134]: Sorry; hadn't pushed that update from my local yet, it's updated now.

File: proto/viam/app/v1/robot.proto
Comment (biotinker) [reply proto/viam/app/v1/robot.proto:138]: > Frame system decomposition into proto messages currently simply ignores geometry information. This will be updated to include geometry information.

It was mentioned in text, but the proto message itself was not explicitly mentioned.

This is separate from the other changes we've been discussing, as it's part of the App API, not the RDK API.

Information is captured here: https://viam.atlassian.net/browse/DATA-874

Adding this was missed when Geometries were added to the Viam cloud app

File: proto/viam/common/v1/common.proto
Comment (raybjork) [inline proto/viam/common/v1/common.proto:134]: If we're going this route, it should be `physical_object` as `Geometry` has no way of being repeated

Comment (biotinker) [reply proto/viam/common/v1/common.proto]: done

**Pull request merged**
